VENDORING.md

```diff
@@ -0,0 +1,43 @@
+# Vendoring policies
+
+This document outlines recommended Vendoring policies for Docker repositories.
+(Example, libnetwork is a Docker repo and logrus is not.)
+
+## Vendoring using tags
+
+Commit ID based vendoring provides little/no information about the updates
```

Comment:
We should track both the commit id and the tag, since the tag may change. We may also want to keep some key information for tags that are signed, but that may be too much work to get into.